TrkFilters/src/HelixFilter_module.cc

```diff
@@ -292,9 +296,35 @@ namespace mu2e
         }
       }
     }
+
+    bool dt_range = false;
+    if (_maxDt0 < 0){
+      dt_range = true;
+    }
+    else{
+      for(auto ihs = hscol->begin();ihs != hscol->end(); ++ihs) {
+        auto const&  hel0 = ihs;
+        float  hel0_t0 =  hel0->t0()._t0;
+
+        for(auto ihs = hscol->begin();ihs != hscol->end(); ++ihs) {
+          auto const&  hel1 =  ihs;
+          float hel1_t0 = hel1->t0()._t0;
+          float dt      = hel1_t0 - hel0_t0;
+          if (dt < _maxDt0 && dt > -_maxDt0) {dt_range = true; }
+        }
+      }
+    }
+
+
     evt.put(std::move(triginfo));
     if (!_noFilter){
-      return (nGoodHelices >= _minNHelices);
+      if( _minNHelices > 1){
+        return ((nGoodHelices >= _minNHelices) && (dt_range = true));
```

Comment:
I believe this line assigns `dt_range = true`, perhaps:
`return (nGoodHelices >= _minNHelices) && dt_range`